### PR TITLE
DH-1473 add application version to meta tag

### PR DESCRIPTION
**Pull request by feedmypixel**
Add the application version to a meta tag so its easy to work out which version of the frontend you are currently looking at

### CODE REVIEW

File: src/templates/_layouts/dit-base.njk [hunk 1]
```diff
@@ -5,6 +5,7 @@
     {% block head %}
       <meta http-equiv="content-type" content="text/html; charset=UTF-8">
       <meta name="viewport" content="width=device-width, initial-scale=1">
+      <meta name="app-version" content="{{ APP_VERSION }}">
```
Comment (teneightfive) [inline src/templates/_layouts/dit-base.njk]: Might be nice to prefix these. GOV.UK does something similar and helps distinguish which ones are custom the app.

Maybe:

```html
<meta name="datahub:version" content="{{ APP_VERSION }}">
```

Comment (ztolley) [reply src/templates/_layouts/dit-base.njk]: Actually I like version as it is. A version is generic, it's not unique to datahub.

Comment (ztolley) [reply src/templates/_layouts/dit-base.njk]: If it was attributes about datahub, like the current entity name or something then I would prefix them

Comment (teneightfive) [reply src/templates/_layouts/dit-base.njk]: But it's the version of the datahub rendering application, no?

If you don't like the idea of prefixing things specific to the frontend app I think just `version` is enough.

Comment (ztolley) [reply src/templates/_layouts/dit-base.njk]: so just go with version and we can all agree with that

**Comment by feedmypixel** [comment]
@teneightfive @ztolley have changed to `<meta name="version" content="{{ APP_VERSION }}">`

**Pull request merged**